src/code.js

```diff
@@ -206,14 +206,15 @@ ${slugs
       })
     }
   }
-  
+
   class BodyRewriter {
     constructor(SLUG_TO_PAGE) {
       this.SLUG_TO_PAGE = SLUG_TO_PAGE;
     }
     element(element) {
       element.append(\`<div style="display:none">Powered by <a href="http://fruitionsite.com">Fruition</a></div>
       <script>
+      window.CONFIG.domainBaseUrl = "https://\${MY_DOMAIN}";
```

Comment:
Apologies for the noise (my IDE cleaned up the whitespace). Here is the change for `code.js`